README.md

```diff
@@ -68,6 +69,7 @@ After writing this method, our new result should be:
 ```
 
 **think**: Where should we call `strip_punctuation`? Does it matter? Why?
+We should call it before we start looping through the array.  That way it will deal with all punctuation in the string at once.
```

Comment:
This is true. I more meant, should you call strip_punctuation inside or outside the analyze_mood method.